evennia/utils/ravensgleaning.py

```diff
@@ -0,0 +1,441 @@
+"""
+This is a Python re-implementation of the ANSI Parser "RavensGleaning" written in JavaScript.
+
+It converts raw ANSI strings to HTML spans.
+
+# TODO: Add proper accreditation to the original Author, @PresJPolk
```

Comment:
We need proper licensing and accreditaion, yes. Also licensing, if any, needs to be made clear.